File: tmp_seo_update.py
import os

file_path = 'index.html'
if not os.path.exists(file_path):
    print(f'File {file_path} not found')
    exit(1)

with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

# Meta Update
old_meta = "Premium windows, doors, and siding in Grimsby & Hamilton. Licensed, insured, and budget-friendly. Get your free estimate today!"
new_meta = "Apex Windows & Exteriors — Grimsby's #1 for Windows, Doors & Siding. We PRICE MATCH & BEAT any quote. Licensed & Insured. Serving Grimsby, Hamilton & Niagara."
content = content.replace(old_meta, new_meta)

# Sticky CTA Injection
sticky_html = """
    <!-- STICKY CONVERSION BANNER (Acquisition Upgrade) -->
    <div class="sticky-cta">
        <div class="sticky-details">
            <span class="sticky-tag">PRICE MATCH GUARANTEE</span>
            <span class="sticky-msg">We WILL beat any competitor quote!</span>
        </div>
        <a href="tel:6478334262" class="sticky-btn">
            <span>📞 Free Estimate: 647-833-4262</span>
        </a>
    </div>
"""

if 'sticky-cta' not in content:
    # Use lowercase bodies since some tools or minifiers might change it, but standard is fine
    if '</body>' in content:
        content = content.replace('</body>', sticky_html + '</body>')
    elif '</BODY>' in content:
        content = content.replace('</BODY>', sticky_html + '</BODY>')

with open(file_path, 'w', encoding='utf-8') as f:
    f.write(content)

print('Success: Apex Acquisition Update v2.0 Applied.')

File: tmp_seo_update.js
const fs = require('fs');
const file = 'index.html';

try {
    let content = fs.readFileSync(file, 'utf8');

    // SEO Meta
    const oldMeta = "Premium windows, doors, and siding in Grimsby & Hamilton. Licensed, insured, and budget-friendly. Get your free estimate today!";
    const newMeta = "Apex Windows & Exteriors — Grimsby's #1 for Windows, Doors & Siding. We PRICE MATCH & BEAT any quote. Licensed & Insured. Serving Grimsby, Hamilton & Niagara.";
    content = content.replace(oldMeta, newMeta);

    // Sticky CTA
    const sticky = `
    <!-- STICKY CONVERSION BANNER (Acquisition Upgrade) -->
    <div class="sticky-cta">
        <div class="sticky-details">
            <span class="sticky-tag">PRICE MATCH GUARANTEE</span>
            <span class="sticky-msg">We WILL beat any competitor quote!</span>
        </div>
        <a href="tel:6478334262" class="sticky-btn">
            <span>📞 Free Estimate: 647-833-4262</span>
        </a>
    </div>
    `;

    if (!content.includes('sticky-cta')) {
        if (content.includes('</body>')) {
            content = content.replace('</body>', sticky + '</body>');
        } else if (content.includes('</BODY>')) {
            content = content.replace('</BODY>', sticky + '</BODY>');
        }
    }

    fs.writeFileSync(file, content);
    console.log('v2.0 Acquisition Update Successfully Applied to index.html');
} catch (err) {
    console.error('Failed to update index.html:', err.message);
    process.exit(1);
}
